lib/datasource/rubygems/get.ts

```diff
@@ -52,7 +52,7 @@ export async function getDependency(
 
   const versions = (await fetch(dependency, registry, VERSIONS_PATH)) || [];
```

Comment:
Doesn't this request *throw* ? If so then everything below it is not executed.

Reply:
i thought it doesnt because of the ||  if not than i can add a try catch around it

Reply:
just to be sure a try catch would look like the following right? I can implement this if you want me to.

```js
let versions = []
try {
 versions = await fetch(dependency, registry, VERSIONS_PATH);
} catch {
// all of my code
}
```